Misc/NEWS.d/next/Core and Builtins/2021-12-24-20-21-45.bpo-46055.R0QMVQ.rst

```diff
@@ -1,2 +1,3 @@
-Speed up shifting operation involving integers less than
-:c:macro:`PyLong_BASE`. Patch by Xinhang Xu.
+Speed up integer shifts: faster left and right shifts of integers smaller than
```

Comment:
The intent is that this news entry covers PRs #30044, #30243 and #30277 (this one).

Reply:
There have been releases since #30044 and #30243, so we'll need a new news entry.